RemotingSample/Packages/packages-lock.json

```diff
@@ -164,23 +155,21 @@
       "url": "https://packages.unity.com"
     },
     "com.unity.xr.arfoundation": {
-      "version": "4.2.3",
+      "version": "4.1.7",
```

Comment:
Seems like we are downgrading the version of this and com.unity.xr.arsubsystems. Is that intended?

Reply:
I think some of these manifest was modified by unity 2021 but the project later was downgraded to unity 2020 and therefore left over some higher versions.